t/porting/regen.t

```diff
@@ -2,6 +2,11 @@
 
 # Verify that all files generated by perl scripts are up to date.
 
+# Hint:  A failure in this file can often be corrected by running:
+#   ./perl -Ilib regen/_________.pl
+# ... where you fill in the blank by looking at what 'make test_porting'
+# tells you.
+
```

Comment:
A ::diag or ::note may show that to the tester, so they don't have to dig in the code

Reply:
I agree and to be honest this do not belong to this branch at all
going to create a new PR for this single change

Reply:
here is a suggestion to fix this #18030